lib/massmailer.js

```diff
@@ -46,7 +47,7 @@ exports.sendAll = async (req, res) => {
         // TODO: Think, maybe use Pug or EJS for that?
         // TODO: Think what else will we need? Probably remove after Agora Bucuresti if there
         // won't be something required.
-        const email = application.email;
+        const notificationEmail = await core.getMember(req, application.user_id).notification_email;
```

Comment:
this one will do like 800 requests to the core each time someone would want to use a massmailer, what I'd do instead is storing the notification email in the application, the same way and for the same reason body name and email is stored here, then you can send email without even querying core

Reply:
Yeah, but that will not update the notification_email so you might get the wrong one. You normally apply to events a few months in advance so the data is not very recent

Reply:
what do you think of doing it all with 1 query then?

Reply:
That would be awesome, but I wasn't quite sure how to do that. But I haven't given it too much thought yet so with some more time I probably could figure that out.
And figuring out why application creation fails 😅

Reply:
just use /members endpoint of core, it'll return them all, then you can just find the correct user in results. one issue here is that you have to have the global:view:member permission, otherwise the call to core will fail

Reply:
Would that actually be better? Then we would get a much bigger query back and I am unsure if that will be any quicker in the end. Having the global:view:member permission might not be needed if we use the superadmin information from config.

Reply:
depends on how much emails needs to be send, but in general, 1 big query is better than 500 smaller ones

Reply:
but this was also the reason why I added email to application, to reduce it to 0 and to not bother with permissions

Reply:
Yeah, adding the emails to applications is indeed a nice solution but does not work for fields that can change somewhere else unfortunately

Reply:
I've removed this out of this PR and made a TODO for it. I do want to merge the other changes since those features are broken currently.